control_region_2016UL.py

```diff
@@ -0,0 +1,921 @@
+                #     "mc_trigger_sf": "Trg_IsoMu27_pt_eta_bins",
+                #     "mc_muon_trg_extrapolation": 1.0,  # for nominal case
+                # },
+                # {
+                #     "flagname": "trg_wgt_single_mu22ormu27",
+                #     "mc_trigger_sf": "Trg_IsoMu27_or_IsoMu22_pt_eta_bins",
+                #     "mc_muon_trg_extrapolation": 1.0,  # for nominal case
+                # },
+            ]
+        },
+    
+    )
+
+    # # muon trigger SF settings from embedding measurements
+    configuration.add_config_parameters(
+        [ "mm"],
+        {
+            "singlemuon_trigger_sf": [
+                {
+                    "flagname": "trg_wgt_single_mu22",
+                    "embedding_trigger_sf": "Trg_pt_eta_bins",
+                    "muon_trg_extrapolation": 1.0,
+                },
+                # {
+                #     "flagname": "trg_wgt_single_mu27",
+                #     "embedding_trigger_sf": "Trg_IsoMu27_pt_eta_bins",
+                #     "muon_trg_extrapolation": 1.0,
+                # },
+                # {
+                #     "flagname": "trg_wgt_single_mu24ormu27",
+                #     "embedding_trigger_sf": "Trg_IsoMu27_or_IsoMu22_pt_eta_bins",
+                #     "muon_trg_extrapolation": 1.0,
+                # },
+            ]
+        },
+    )
+
+    
+
+    # add muon scalefactors from embedding measurements
+    configuration.add_config_parameters(
+        [ "mm"],
+        {
+            "mc_muon_sf_file": EraModifier(
+                {
+                    "2016preVFP": "data/embedding/muon_2016preVFPUL.json.gz",
+                    "2016postVFP": "data/embedding/muon_2016postVFPUL.json.gz",
+                    "2017": "data/embedding/muon_2017UL.json.gz",
+                    "2018": "data/embedding/muon_2018UL.json.gz",
+                }
+            ),
+            "mc_muon_id_sf": "ID_pt_eta_bins",
+            "mc_muon_iso_sf": "Iso_pt_eta_bins",
+            "mc_muon_id_extrapolation": 1.0,  # for nominal case
+            "mc_muon_iso_extrapolation": 1.0,  # for nominal case
+        },
+    )
+
+    # add muon scalefactors from embedding measurements
+    configuration.add_config_parameters(
+        ["mt", "mm", "em"],
+        {
+            "embedding_muon_sf_file": EraModifier(
+                {
+                    "2016preVFP": "data/embedding/muon_2016preVFPUL.json.gz",
+                    "2016postVFP": "data/embedding/muon_2016postVFPUL.json.gz",
+                    "2017": "data/embedding/muon_2017UL.json.gz",
+                    "2018": "data/embedding/muon_2018UL.json.gz",
+                }
+            ),
+            "embedding_muon_id_sf": "ID_pt_eta_bins",
+            "embedding_muon_iso_sf": "Iso_pt_eta_bins",
+        },
+    )
+
+    configuration.add_config_parameters(
+        ["mm"],
+        {
+            "doublemuon_trigger": EraModifier(
+                {
+                    "2018": [
```

Comment:
I would rename the config to be something like `dimuon_control_region.py` and check, that it can be used for not only 2016, but also the other eras (probably is already the case)